lib/news_scraper/constants.rb

```diff
@@ -1,7 +1,7 @@
 module NewsScraper
   module Constants
-    TEMP_DIRS = YAML.load_file('config/temp_dirs.yml')
-    SCRAPE_PATTERN_FILEPATH = 'config/article_scrape_patterns.yml'.freeze
```

Comment:
freeze isn't reallllly needed, and it makes overriding in tests difficult. I can't think of another way to override this otherwise